code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -139,6 +139,14 @@
 	build_path = /obj/item/reagent_containers/glass/bottle/nutrient/empty
 	category = list("initial", "Botany Chemicals")
 
+/datum/design/soil
+	name = "Clump of soil"
```

Comment:
```suggestion
	name = "Clump of Soil"
```
Its not standardized on the biogenerator but usually we have it like this in other places.

Reply:
will do